.env

```diff
@@ -3,8 +3,6 @@ DB_PORT="5432"
 DB_USER="root"
 DB_PASSWORD="password"
 DB_NAME="boardly"
-
-SERVER_HOST="localhost"
+SERVER_HOST="127.0.0.1"
```

Comment:
Also is there a reason why you are not using `localhost `here instead?

Reply:
no particular reason, just me when i was debugging 
i will change it to the IP address now